usaon_benefit_tool/templates/base.html

```diff
@@ -94,6 +94,13 @@
         return new bootstrap.Tooltip(tooltipTriggerEl)
       })
     </script>
+
+    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
```

Comment:
Is JQuery a required dependency of DataTables? These days there's `document.querySelector()` which can do lots (all?) of the same selection magic as JQuery, so I've been getting away without it.

What do you think?

Reply:
It seems like JQuery is a required depedency of DataTables however the website provides an example on how you can sort a table without calling into JQuery (it is still a dependency and has to be loaded though) 
https://datatables.net/examples/non_jquery/init.html

Reply:
Actually, https://github.com/fiduswriter/Simple-DataTables this might just do the job.

Reply:
Actually I have found an alternative solution where jQuery wont be required. It can be found in the following link https://fiduswriter.github.io/simple-datatables/documentation/

Reply:
Good find! I really like the idea of using a lighter-weight library. Do you think that would be an improvement as well?

Please don't feel obligated to change, I just wanted to make sure JQuery was required for DataTables if we are including it :)

Reply:
I think we should go for a solution with the lighter-weight library. I feel it would be better choice although it looks a bit different from DataTables with more spaces between the sorting buttons. but yeah if we have an option of choosing a lighter-weight library then maybe we should proceed with it .

Reply:
Right on, let's do it! Thanks for your input :)